index.js

```diff
@@ -15,3 +15,179 @@ hamburger.addEventListener('click', show);
+    listItem3: 'Ruby on rails',
+    image: 'images/Calender.png',
+    liveLink: '#',
+    sourcelink: 'https://github.com/Yerimah/My-portfolio.git',
+
+  },
+];
+
+const projectButton = document.querySelectorAll('.card-btn');
+const windowContainer = document.createElement('div');
+const container = document.createElement('div');
+function closeIt() {
+  document.body.removeChild(windowContainer);
+  window.location.reload();
+}
+
+projectButton.forEach((el) => el.addEventListener('click', () => {
+  const displayedArray = cardWorks.filter((projectId) => projectId.id === el.getAttribute('id'));
+
+  for (let i = 0; i < displayedArray.length; i += 1) {
+    document.body.appendChild(windowContainer);
+    windowContainer.classList.add('popup-window');
+    windowContainer.appendChild(container);
+    container.classList.add('pop-up');
+    const img = document.createElement('img');
+    img.classList.add('pop-image');
+    img.setAttribute('src', displayedArray[i].image);
+
+    container.appendChild(img);
+
+    const title = document.createElement('h3');
+    title.classList.add('pop-title');
+    title.textContent = displayedArray[i].name;
+    container.appendChild(title);
+    const headContainer = document.createElement('div');
+    container.insertBefore(headContainer, img);
+    headContainer.className = 'headContainer';
+
+    const ul = document.createElement('ul');
+    ul.classList.add('pop-list');
+    container.appendChild(ul);
+    headContainer.append(title, ul);
+    const listItem1 = document.createElement('li');
+    listItem1.textContent = displayedArray[i].listItem1;
+    listItem1.classList.add('pop-cell');
+    const listItem2 = document.createElement('li');
+    listItem2.textContent = displayedArray[i].listItem2;
+    listItem2.classList.add('pop-cell');
+    const listItem3 = document.createElement('li');
+    listItem3.textContent = displayedArray[i].listItem3;
+    listItem3.classList.add('pop-cell');
+    ul.appendChild(listItem1);
+    ul.appendChild(listItem2);
+    ul.appendChild(listItem3);
+
+    const description = document.createElement('p');
+    description.classList.add('pop-description');
+    description.textContent = displayedArray[i].description;
+    container.appendChild(description);
+    const btnTxtContainer = document.createElement('div');
+    container.appendChild(btnTxtContainer);
+    btnTxtContainer.className = 'btn-text';
+
+    const imgTxtContainer = document.createElement('div');
+    container.appendChild(imgTxtContainer);
+    imgTxtContainer.className = 'img-text';
+
+    const linkContainer = document.createElement('div');
+    linkContainer.classList.add('link-container');
+    container.appendChild(linkContainer);
+
+    const liveLink = document.createElement('a');
+    const linkText1 = document.createTextNode('See Live');
+    const liveIcon = document.createElement('img');
+    liveIcon.classList.add('popIcon');
+    liveIcon.src = 'images/livepop.png';
+    liveIcon.alt = ' ';
```

Comment:
- [ ] I suggest you provide the `alt` text. This is a useful way of helping visually impaired users who might be needing the help of a screen reader to read the images on the page.